src/main/java/ca/ciccc/Assignment2.java

```diff
@@ -33,9 +51,20 @@ public String numberLoops(int n) {
      * @return true if the number of 1s is greater than the number of 7s
      */
     public boolean countNumbers(int[] nums) {
-        // Todo: your code goes here
+        int numberOfOnes = 0;
+        int numberOfSevens = 0;
 
-        return false;
+        for (int i = 0; i < nums.length; ++i) {
+            if (nums[i] == 1) {
+                numberOfOnes += 1;
+            }
+
+            if (nums[i] == 7) {
+                numberOfSevens += 1;
+            }
+        }
+
+        return numberOfOnes > numberOfSevens;
```

Comment:
This one is the same return true if the number of 1s is greater than the number of 7s

Reply:
Thanks. However, this is the same as https://github.com/itzderr/Assignment2-Java/pull/2#discussion_r260456811.

Reply:
yes, it seems to be ok.